### ISSUE TITLE
[BUG] "This app isn't available for your device because it was made for an older version of Android" on version 13

### ISSUE THREAD

**Issue by ValJed** [zeapo/Android-Password-Store#2764]
### Describe the bug

Hi!
Just switched my password manager to use password store in order to avoid third party services storing my passwords.
The only limitation I'm facing here is that this app isn't available on my android version which is the 13.
I'm not familiar with mobile development as well as android dev. Not sure what makes an app unavailable to newer versions.
As I said I'm not familiar with mobile development but maybe I can help with indications.

Thank you for your feedback.

EDIT: Just saw [this issue](https://github.com/android-password-store/Android-Password-Store/issues/2316)
about a feature not working properly on android 13. I'm wondering why I can't install the app on my android 13 device?
### Steps to reproduce

Try to install the app on a phone running on Android 13 through google play.

### Expected behavior

App available on Android 13 through google play.

### Device information

- Device: Samsung S21
- OS: Android 13

**Comment by msfjarvis** [comment]
Duplicate of #2669, #2663, https://github.com/android-password-store/Android-Password-Store/issues/2644, https://github.com/android-password-store/Android-Password-Store/issues/2587, https://github.com/android-password-store/Android-Password-Store/issues/2553, https://github.com/android-password-store/Android-Password-Store/issues/2533 and https://github.com/android-password-store/Android-Password-Store/issues/2417.

TL;DR Android places limits on how old a version of Android apps can target (for your safety) and Password Store's Play Store builds run afoul of that limit. Use the snapshot builds https://docs.passwordstore.app/docs/users/release-channels

**Comment by ValJed** [comment]
Sorry for the duplicate issue, thank you!

**Comment by jidhub** [comment]
Ok, this is why the Android-Password-Store version on f-droid linked from https://passwordstore.app is three years old.

@msfjarvis you might want to add the [above sentence](https://github.com/android-password-store/Android-Password-Store/issues/2764#issuecomment-1812644873) to https://passwordstore.app and to remove f-droid and google-play links from its top bar.
